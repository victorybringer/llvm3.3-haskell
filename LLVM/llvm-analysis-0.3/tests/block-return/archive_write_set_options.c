extern int a_fo(void);
extern int a_co(void);

int archive_write_set_options(void) {
  int r1, r2;

  r1 = a_fo();
  if(r1 < -20)
    return r1;

  r2 = a_co();
  if(r2 < -20)
    return r2;

  if(r1 == -20 && r2 == -20)
    return -20;

  return 0;
}
