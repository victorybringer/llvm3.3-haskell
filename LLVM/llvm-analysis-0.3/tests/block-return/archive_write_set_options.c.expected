fromList [ ("%if.end8", 0), ("%if.then7", -20) ]